src/main/java/com/example/bookstore/controller/BookController.java

```diff
@@ -37,14 +38,14 @@ public BookDto getBookById(@PathVariable Long id) {
 
     @ResponseStatus(HttpStatus.CREATED)
     @PostMapping
-    public BookDto createBook(@RequestBody CreateBookRequestDto bookDto) {
+    public BookDto createBook(@RequestBody @Valid CreateBookRequestDto bookDto) {
         return bookService.save(bookDto);
     }
 
     @ResponseStatus(HttpStatus.OK)
     @PutMapping("/{id}")
     public BookDto updateBookById(@PathVariable Long id,
-                              @RequestBody CreateBookRequestDto requestDto) {
+                              @RequestBody @Valid CreateBookRequestDto requestDto) {
```

Comment:
Also, it's better to add `@PathVariable @Positive Long id` to getBookById method.
Don't forget to add `@Validated` under the controller